python_ta/contracts/__init__.py

```diff
@@ -55,35 +85,36 @@ def check_type_annotation(annotation: Optional[type], v: Any) -> bool:
             return True
 
 
-def parse_preconditions(docstring: str) -> List[str]:
-    """Return a list of preconditions parsed from the given docstring.
+def parse_conditions(docstring: str, parse_token: str = "Precondition") -> List[str]:
```

Comment:
Reminder: "conditions" -> "assertions", and remember to change the local variables tool.